container-se-managed/src/main/java/org/jboss/arquillian/container/se/managed/ManagedSEContainerConfiguration.java

```diff
@@ -30,10 +30,19 @@
     private Level logLevel = Level.INFO;
     private boolean keepDeploymentArchives = false;
     private String additionalJavaOpts;
+    private int waitTime  = 5;
 
     public void validate() throws ConfigurationException {
     }
 
+    public void setWaitTime(int waitTime) {
+        this.waitTime = waitTime > 0 ? waitTime : 5;
```

Comment:
This could be checked somewhere else (however it is likely bit paranoid and unnecessary) but definitely not in setter method.